doc/overrides/main.html

```diff
@@ -1,13 +1,41 @@
 {% extends "base.html" %}
 
 {% block extrahead %}
-  <!-- Google Tag Manager -->
-  <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-  new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-  j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-  'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-  })(window,document,'script','dataLayer','GTM-PGZQGKM');</script>
-  <!-- End Google Tag Manager -->
+  <script> 
+    rudderanalytics=window.rudderanalytics=[];for(var methods=["load","page","track","identify","alias","group","ready","reset","getAnonymousId","setAnonymousId"],i=0;i<methods.length;i++){var method=methods[i];rudderanalytics[method]=function(a){return function(){rudderanalytics.push([a].concat(Array.prototype.slice.call(arguments)))}}(method)}rudderanalytics.load("1iJtD7UiXR23JlGPtE8YnzQH3KG","https://pachyderm-dataplane.rudderstack.com"),rudderanalytics.page();
```

Comment:
This looks like it has the "Docs" write key. Is that intended, or does it need a different one?

Reply:
Yes, that's the intention. I should've mentioned that this was the docs site integration.